### DISCUSSION TITLE
Have issue after update @@

### DISCUSSION THREAD

**Discussion by nhannt992k** [Codium-ai/codiumai-vscode-release#156] [Q&A]
![image](https://github.com/Codium-ai/codiumai-vscode-release/assets/161809349/cd9fec5b-11ea-460f-a259-be1f0a650397)
 I'm try find css for my code and meet this case! How to fix this @@ Tks for help!!

**Comment by almog-lv** [comment]
Hey @nhannt992k can you please paste here the request ID?

**Reply by nhannt992k** [reply]
Yes! 424a430a-b78b-4fa7-91c8-02d83c35b0d8

**Comment by oranshiner** [comment]
Hey @nhannt992k Thank you for reporting this issue, we would fix it and let you know when the fix is released

**Comment by nhannt992k** [comment]
Tks :3

Vào 12:58, CN, 7 thg 4, 2024 oranshiner ***@***.***> đã viết:
